pkg/tests/end_to_end_tests/telemetry_test.go

```diff
@@ -32,6 +33,17 @@ func setTobsEnv(prop string) error {
 	return os.Setenv(fmt.Sprintf("TOBS_TELEMETRY_%s", prop), prop)
 }
 
+type e2eTelemetry interface {
```

Comment:
Please add some comments so it's more obvious why is this needed: `we want to manually trigger Sync for tests..`

Reply:
Also consider better naming: eg `testTelemetryEngine`

Reply:
> Also consider better naming: eg testTelemetryEngine

That name sounds like name for a test function